### PR TITLE
Update work action menu with transfer link.

### PR DESCRIPTION
**Pull request by grosscol**
Update new transfers view for GenericWork.
Update locales strings for transfers of works
Move routes for transfers from GenericFile to GenericWork
Update proxy deposit routes spec for GenericWorks.
Add ownership transfer feature specs for Works.
Refactor Transfer Work Action link_to as block.

### CODE REVIEW

File: app/views/my/_work_action_menu.html.erb [hunk 1]
```diff
@@ -28,13 +28,13 @@
         <span> TODO: Highlight Work on Profile </span>
       <% end %>
     </li>
-
     <li role="menuitem" tabindex="-1">
-      <%= link_to('#') do %>
-        <i class="glyphicon glyphicon-transfer" aria-hidden="true"></i>
-        <span> TODO: Transfer Ownership of Work </span>
+      <% gw = document.to_model %>
```
Comment (cam156) [inline app/views/my/_work_action_menu.html.erb]: @grosscol Do we need the to_model here?  That requires the work being loaded from fedora.

File: spec/features/ownership_transfer_spec.rb [hunk 2]
```diff
@@ -32,32 +32,66 @@
...
       context 'If the new owner accepts it' do
-      pending 'The transfer option is not available' do
-        fail
-      end
+        before do
+          new_owner.proxy_deposit_requests.last.transfer!
+          user_utility_toggle.click
+          click_link 'transfer requests'
+        end
+        specify 'I should see it was accepted' do
```
Comment (cam156) [inline spec/features/ownership_transfer_spec.rb]: @grosscol is there a reason you are using specify here and it above?

Comment (grosscol) [inline spec/features/ownership_transfer_spec.rb]: Probably the flavor that came along with the copy pasta.  Correcting specify calls to it calls.

File: app/views/my/_work_action_menu.html.erb (hunk 1 above)
Comment (grosscol) [inline app/views/my/_work_action_menu.html.erb]: @cam156 It does not appear that the model is required.  Removing.

**Pull request merged**